### PR TITLE
Add installation instruction on README

### PR DESCRIPTION
**Pull request by kitchoi**
This PR updates the README for Pyface (also visible on PyPI page) to provide installation instructions for optional dependencies.

### CODE REVIEW

**Review by rahulporuri — APPROVED** [review]
LGTM

File: README.rst [hunk 1]
```diff
@@ -24,6 +24,28 @@ The following GUI backends are supported:
...
+
+To install with PySide2 dependencies::
+
+    $ pip install pyface[pyside2]
+
+To install with PyQt5 dependencies::
+
+    $ pip install pyface[pyqt5]
+
+To install with wxPython dependencies (experimental)::
```
Comment (corranwebster) [inline README.rst:41]: Probnably should specify wxPython 4?

File: README.rst [hunk 2]
```diff
@@ -56,4 +78,4 @@ follow the instructions in ``etstool.py``.  In particular::
 
     > python etstool.py test-all
 
-will run tests in all supported environments automatically.
\ No newline at end of file
+will run tests in all supported environments automatically.
```
Comment (kitchoi) [inline README.rst:81]: Ah... I tried to change 'wxPython' to 'wxPython4' via GitHub directly and GitHub did this change for me!

Comment (kitchoi) [reply README.rst:81]: This adds the missing newline at the end of file, and I am going to merge away...

**Pull request merged**